.editorconfig

```
@@ -0,0 +1,27 @@
+root = true
+
+[*]
+indent_style = space
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+indent_size = 2
+max_line_length = 110
+
+[*.{js,ts}]
+block_comment_start = /*
+block_comment_end = */
+
+[{en.js,es.js,*.sh,*.md,Dockerfile,.editorconfig,package.json}]
+max_line_length = 220
+
+# Unset for binary and generated files
+[{**.svg,**.png,**/obj/**,**/bin/**,.vscode/**,**/Properties/launchSettings.json,*.[sS]ecret.*,yarn.lock,.doppler-ci,build/**}]
```

Ya que entiendo son propias de NET, las podríamos eliminar de los _mfe_, no?
```suggestion
[{**.svg,**.png,.vscode/**,**/Properties/launchSettings.json,*.[sS]ecret.*,yarn.lock,.doppler-ci,build/**}]
```

---

Según lo acordado con @ms-darianbenito, llegamos a la conclusión de que estaría bien hacer esta modificación para proyectos Micro-Frontend que usen esta estandarización/template CI/CD. Pero, por motivos de no modificar el Scope actual, se tendrá en cuenta como un refactor de dicho 'template'.
Cc: @hrocca, @andresmoschini

---

Yo creo que se podría dejar sin problemas, no le hace mal a nadie que se ignoren esas carpetas.